src/midi/midi_lasynth_model.cpp

```diff
@@ -94,20 +112,26 @@ bool LASynthModel::Load(const service_t& service, const std_fs::path& dir) const
 	                                 mt32emu_return_code expected_code) -> bool {
 		if (!rom_full)
 			return false;
-		const auto rom_path = dir / rom_full->filename;
-		const auto rcode = service->addROMFile(rom_path.string().c_str());
+		const auto rom_path = GetROMPath(dir, rom_full->filename);
+		if (!rom_path)
+			return false;
+		const auto rcode = service->addROMFile(rom_path->string().c_str());
 		return rcode == expected_code;
 	};
 
 	auto load_both = [&service, &dir](const Rom *rom_1, const Rom *rom_2,
 	                                  mt32emu_return_code expected_code) -> bool {
 		if (!rom_1 || !rom_2)
 			return false;
-		const auto rom_1_path = dir / rom_1->filename;
-		const auto rom_2_path = dir / rom_2->filename;
+		const auto rom_1_path = GetROMPath(dir, rom_1->filename);
+		if (!rom_1_path)
+			return false;
+		const auto rom_2_path = GetROMPath(dir, rom_2->filename);
+		if (!rom_2_path)
+			return false;
```

Comment:
Thanks!

Do you have `clang-format` in your path? 

If you can run `./scripts/format-commit.sh`, it will update the formatting of just the changed lines.

Reply:
Yes, I  just ran that script and pushed a new commit with fixed formatting.